examples/sn-gatsby-example/gatsby-config.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  siteMetadata: {
+    title: 'sn-gatsby-example',
+    description: 'A blog app with gatsby and typescript',
+    keywords: 'gatsby, javascript, sensenet, typescript',
+    siteUrl: 'https://github.com/SenseNet/sn-client.com',
+    author: {
+      name: 'Sense/Net',
+      url: 'http://sensenet.com/',
+      email: 'snteam@sensenet.com',
+    },
+  },
+  plugins: [
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        name: 'content',
+        path: `${__dirname}/src/content`,
+      },
+    },
+    {
+      resolve: 'gatsby-transformer-remark',
+      options: {
+        plugins: [
+          {
+            resolve: 'gatsby-remark-responsive-iframe',
+            options: {
+              wrapperStyle: 'margin-bottom: 1rem',
+            },
+          },
+          'gatsby-remark-prismjs',
```

Comment:
does syntax highlighting work?

Reply:
I had to add a theme (to gatsby-browser.js), now it is working require('prismjs/themes/prism-solarizedlight.css')